Fix/typos

DanielLitvak opened the pull request:
fixes #55

lmilbaum commented on git/README.md:
```diff
@@ -15,7 +15,6 @@
     </details>
 * commit 
 * push
-* push
```
Previous PR already addressed this issue. This is a duplicate

lmilbaum commented:
@DanielLitvak why did you close this PR?

DanielLitvak commented:
Didn't mean to.
1) I deleted the branch.
2) pulled to my master.
3) created a new branch with the same name.
4) fixed the issue.
5) created a new pull request.

guess I could have just pulled to the branch and fix the issue. Next time I will know better.
